Dockerfile

```diff
@@ -15,8 +15,7 @@ CMD ["poetry", "run", "flask", "run", "-h", "0.0.0.0"]
 FROM base AS prod
 COPY . /app
 WORKDIR /app
-EXPOSE 5000
-ENV PORT=${PORT}
+ENV PORT=5000
```

Comment:
Your readme instructions currently assume port 8000 rather than 5000 - worth noting that this means that those instructions no longer work

Reply:
Fix in 604a331284038726a07172d074134b4351701301